step-certificates/examples/certificate_authority_single_instance/generate-values.sh

```diff
@@ -0,0 +1,105 @@
+ROOT_CA_NAME=`jq -r '.root_ca_name' ca.config`
+INTERMEDIATE_CA_NAME=`jq -r '.intermediate_ca_name' ca.config`
+CA_ORG_NAME=`jq -r '.ca_org_name' ca.config`
+CA_COUNTRY_NAME=`jq -r '.ca_country_name' ca.config`
+CA_LOCALITY_NAME=`jq -r '.ca_locality_name' ca.config`
+CA_DNS_NAMES=`jq -c .ca_dns_names ca.config`
+CA_URL=`jq -r .ca_url ca.config`
+JWK_PROVISIONER_NAME=`jq -r .jwk_provisioner_name ca.config`
+
+export ROOT_CA_NAME INTERMEDIATE_CA_NAME CA_ORG_NAME CA_COUNTRY_NAME CA_LOCALITY_NAME CA_DNS_NAMES CA_URL JWK_PROVISIONER_NAME
+
+# Write Out Root and Intermediate Certificate Templates
+cat root-tls.json.tpl | envsubst | tee root-tls.json
+cat intermediate-tls.json.tpl | envsubst | tee intermediate-tls.json
+
+# Generate Root and Intermediate Passwords
+ROOT_TLS_PASSWORD_B64=`tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_\`{|}~' </dev/urandom | head -c 64 | tee root-tls.password | base64 --wrap=0`
+INTERMEDIATE_TLS_PASSWORD_B64=`tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_\`{|}~' </dev/urandom | head -c 64 | tee intermediate-tls.password | base64 --wrap=0`
```

Comment:
Make this more compatible
```suggestion
ROOT_TLS_PASSWORD_B64=`LC_ALL=C tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_\`{|}~' </dev/urandom | head -c 64 | tee root-tls.password | step base64`
INTERMEDIATE_TLS_PASSWORD_B64=`LC_ALL=C tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_\`{|}~' </dev/urandom | head -c 64 | tee intermediate-tls.password | step base64`
```